report_templates/subscriptions.erb

```diff
@@ -21,6 +21,10 @@ require:
         'Quantity': pool.quantity,
         'SKU': pool.product_id,
         'Contract number': pool.contract_number,
+        'Account number': pool.account_number,
+        'Start date': pool.start_date,
+        'End date': pool.end_date,
+        'Organization': "org_label_here",
```

Comment:
for this, we need to enable organization in safe mode first, I opened https://github.com/Katello/katello/pull/8532
once merged, you can then use `pool.organization.name`

that also needs updating include in load_pools to avoid SQL inefficiency